### PR TITLE
New strings translated to French

### PR DESCRIPTION
**Pull request by el00ruobuob**
Ready for review @erciccione

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -336,24 +336,24 @@
...
+
+    <string name="menu_language">Langue</string>
+    <string name="language_system_default">Utiliser la Langue du Système</string>
+
+    <string name="receive_desc_hint">Description (optionnelle)</string>
+
+    <string name="send_address_not_openalias">Adresse OpenAlias indisponible</string>
+    <string name="send_address_openalias">Sécurisé OpenAlias &#x2714;</string>
+    <string name="send_address_resolve_openalias">Résolution OpenAlias&#8230;</string>
+    <string name="send_address_no_dnssec">OpenAlias sans DNSSEC - l'adresse pourrait être usurpée !</string>
```
Comment (erciccione) [inline app/src/main/res/values-fr/strings.xml]: You forgot to escape `l'adresse` here

**Review by erciccione — CHANGES_REQUESTED** [review]
Thanks for this, just a small mistake to fix

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (el00ruobuob) [reply app/src/main/res/values-fr/strings.xml]: Damned. I fix this soon.

Comment (el00ruobuob) [reply app/src/main/res/values-fr/strings.xml]: fixed.

**Comment by el00ruobuob** [comment]
It should be good now.

**Review by m2049r — APPROVED** [review]

**Review by erciccione — APPROVED** [review]

**Pull request merged**